the-wing/components
# Mo 554/multi select

## Description
- Added multiselect
- Moved all react-select custom components to their own files
- Moved all stories to be nested under "containers" and "ui" for ease of use

## Jira Ticket
https://thewing.atlassian.net/browse/MO-554

## Screenshots
![screen shot 2019-01-07 at 2 32 44 pm](https://user-images.githubusercontent.com/1829422/50789051-1db91880-1289-11e9-834c-33f0715452ae.png)
![screen shot 2019-01-07 at 2 32 38 pm](https://user-images.githubusercontent.com/1829422/50789052-1e51af00-1289-11e9-8178-2f02fabce211.png)

## How should this be manually tested?
1. Go to "UI > Forms > Select > search"
2. Confirm that everything matches designs for DESKTOP ONLY
3. Go to "UI > Forms > Select > search with many values"
4. Confirm that everything matches designs for DESKTOP ONLY

## Type of change

_Please delete options that are not relevant._


## Checklist:

~- [ ] I have written tests or updated existing tests for this change~ *update when testing harness has been added*

~- [ ] New and existing tests pass locally~ *update when testing harness has been added*
